src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,285 @@
+package Duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+
+    public static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
+
+        Scanner in = new Scanner(System.in);
+
+        try {
+            createFile();
+        } catch (IOException e) {
+            printFileError();
+        }
+        printWelcomeMessage();
+        String userInput = in.nextLine();
+
+        while (!userInput.equals("bye")) {
+
+            //If user input list, return list of items
+            if (userInput.equals("list")) {
+                printList();
+                //If user input done, mark the task as done
+            } else if (userInput.contains("done")) {
+                isCompleted(userInput);
+
+            } else if(userInput.contains("delete")){
+                toDeleteTask(userInput);
+            }
+            else {
+                addTask(userInput);
+            }
+            userInput = in.nextLine();
+        }
+        printExitMessage();
+    }
+
+    public static void printWelcomeMessage() {
+
+
+        String logo = "    ____   ____ " + System.lineSeparator()
+                + "   |  __| |  _ \\ " + System.lineSeparator()
+                + "   | |__  | | | |    " + System.lineSeparator()
+                + "   |  __| | | | |    " + System.lineSeparator()
+                + "   | |__  | |_| |  " + System.lineSeparator()
+                + "   |____| |____/  ";
+
+
+        ;
+        System.out.println("    Hi! I am easy\n" + logo);
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    What can I do for you?");
+        System.out.println("    ____________________________________________________________");
+    }
+
+    private static void printFileError() {
+        System.out.println(" Error reading/writing data.txt.");
+    }
+
+    public static void printList() {
+
+
+        if (tasks.size() == 0) {
+            new Exception("empty list");
+            return;
+        }
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    Here are the tasks in your list:");
+
+        for (Task task : tasks) {
+            System.out.print(tasks.indexOf(task) + 1 + ".");
+            task.printTask();
+        }
+        System.out.println("    ____________________________________________________________");
+    }
+
+
+
+    //completed tasks ( e.g. done 3)
+    public static void isCompleted(String command) {
```

Comment:
The chosen `isCompleted()` name for this function does not seem to be congruent with its logic. Perhaps a better naming would be `completeTask()` or `markTaskAsDone()`?